File: Week_2/problem_8-Replacement.c
// https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/C

#include <stdio.h>

int main()
{
    int n;
    scanf("%d", &n);
    int ar[n];

    for (int i = 0; i < n; i++)
    {
        scanf("%d", &ar[i]);
        // printf("%d ", ar[i]);

        if (ar[i] > 0)
        {
            ar[i] = 1;
        }
        else if (ar[i] < 0)
        {
            ar[i] = 2;
        }
    }

    for (int j = 0; j < n; j++)
    {
        printf("%d ", ar[j]);
    }

    return 0;
}

File: Week_2/problem_8-Replacement.py
# https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/C

n = int(input())
ar = list(map(int, input().split()))

for i in range(n):
    if ar[i] > 0:
        ar[i] = 1
    elif ar[i] < 0:
        ar[i] = 2

print(*ar)
